hack/relayer.Containerfile

```diff
@@ -0,0 +1,13 @@
+FROM ghcr.io/succinctlabs/sp1:v4.1.0 as BUILD
```

Comment:
I am adament about hardcoding this for now, but it sure would be nice if we could find a way to not have the sp1 verison hardcoded here as well.

Reply:
added a variable